scripts/docker-tag.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/sh
+# docker-tag.sh - Create a new tag for a set of Agoric Docker images
+set -xe
+
+SRCTAG=$1
+DSTTAG=$2
+DOCKERUSER=${3-agoric}
+
+if [ -z "$TAG" ]; then
+  echo 1>&2 "Usage: $0 SRCTAG TAG"
```

Comment:
Is `$TAG` an command line argument or is it expected to be set in the environment before running this script?

Reply:
Thanks for catching this!

I tested it, but then renamed some variables.  Tested again now.